File: Assignment5&7/program8.c
//write a program which Accept number from user and return count of odd digits.

#include<stdio.h>

int Countodd(int iNo)
{
    int iDigit = 0;
    int iCnt = 0;
    if(iNo<0)
    {
        iNo = -iNo;
    }

    while(iNo>0)
    {
        iDigit = iNo % 10;
        if((iNo % 2)!=0)
        {
            iCnt++;
        }
        iNo = iNo / 10;
    }
    return iCnt;
}
int main()
{
    int iValue = 0;
    int iRet = 0;
    printf("Enter Number\n");
    scanf("%d",&iValue);

    iRet = Countodd(iValue);
    printf("%d\n",iRet);

    return 0;
}

File: Assinment30&31/program8.java
//write a program which accept one number from user and toggle 7th bit of that number. return modified number


import java.lang.*;
import java.util.*;
//  0000  0000   0000  0000  0000  0000  0100  0000
//  0       0      0    0      0    0      4   0

class Bitwise
{
    public int toggleBit(int iNo)
    {
        int iMask =0X00000040;      //576
        int iResult = 0;

        iResult = iNo ^ iMask;
        return iResult;

    }
}

class program8
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the Number");
        int iValue = sobj.nextInt();

        Bitwise bobj = new Bitwise();
        
        int iRet = bobj.toggleBit(iValue);

        System.out.println("Update Number is : "+iRet);


    }
}